### PR TITLE
Fix typo in smart contract developer docs(Chinese)

### PR DESCRIPTION
**Pull request by yuliyu123**
fix some typo in smart contract lib Chinese translator

### CODE REVIEW

**Comment by joeurassa** [comment]
Is this possible?

**Comment by minimalsm** [comment]
@riely007 reviewing

File: src/content/translations/zh/developers/docs/smart-contracts/libraries/index.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ sidebar: true
 
 ## 前置要求 {#prerequisites}
 
-在我们跳转到智能合约库之前，清楚地了解一个智能合约的构成是一个不错的主意。 如果尚未进行智能合约的了解，请直接点击</[智能合约](/developers/docs/smart-contracts/anatomy/)。
+在我们跳转到智能合约库之前，清楚地了解一个智能合约的构成是一个不错的主意。 如果尚未进行智能合约的了解，请直接点击[智能合约](/developers/docs/smart-contracts/anatomy/)。
```
Comment (Riely007) [inline src/content/translations/zh/developers/docs/smart-contracts/libraries/index.md:12]: thank you! the modified one looks good!

**Comment by Riely007** [comment]
> fix some typo in smart contract lib Chinese translator
> 
> ## Description
> ## Related Issue

Hey thank you for pointing the typo. 😁

1. for the 1st one, you are right! :) 
2. for the 2nd one, the "暂停“ needs to be Italics so i think the original one looks good!

**Comment by joeurassa** [comment]
Where do I need to go for editing?

**Comment by Riely007** [comment]
> Update index.md

looks nice! thank you! :) 

now it's correct  ✅

**Comment by yuliyu123** [comment]
np, it's my pleasure to participate in such a meaningful project. :)

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
@all-contributors please add @yuliyu123 for translations

**Pull request merged**